packages/react-native-tracer-provider/android/build.gradle

```diff
@@ -93,9 +95,8 @@ dependencies {
   // For < 0.71, this will be from the local maven repo
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-native:+"
+  implementation "com.facebook.react:react-android:+"
```

Comment:
should this be left as react-native? Based on the comment above it sounds like the plugin is automatically renaming this on newer versions

Reply:
Covered. Refactored once again the solution and got it working.